tests/trunc-double-uniform.ispc

```diff
@@ -0,0 +1,127 @@
+
+export void f_f(uniform float RET[], uniform float aFOO[]) {
+
+    uniform float error = 0;
+    RET[programIndex] = 0;
+    uniform double deltaLow = 0.000001d;
+    uniform double deltaHigh = 0.999999d;
+    uniform double testVal;
+    uniform double resVal;
+    uniform double baseVal;
+
+    //Case 1 : Generic value 1 < val < +inf with deltaLow.
```

Comment:
I would expect it to be formatted like `// Case 1: bla-bla` - note whitespace after // and no white space before `:`.

Reply:
Changed

Reply:
This is fixed in most of the cases, but not in all. Please go through all the tests and fix it.